tests/testthat/test-grab_names.R

```diff
@@ -0,0 +1,9 @@
+print("a")
```

Comment:
what's this for?

Reply:
Fixed, I put this on initially as a placeholder to save it onto my computer.